### ISSUE TITLE
v1.0.0 attempts to `require_relative` non-existent path

### ISSUE THREAD

**Issue by owst** [rack/rackup#13]
# Problem
Attempting to `require 'rackup'` with v1.0.0 fails with:
```
`require_relative': cannot load such file -- /Users/owenstephens/.rbenv/versions/3.1.3/lib/ruby/gems/3.1.0/gems/rackup-1.0.0/lib/rackup/handler
```

## Reproduction

An example Gemfile:

```
source "https://rubygems.org"

gem "rails", '7.1.2'
gem "redis-rack", "2.1.4" # Depends on Rack < 3
```
after `bundle install`ing, attemping `bundle exec ruby -rrackup -e 'p 42'` errors as above.

## Background

We're midway through upgrading our app to Rails 7.1; we now depend on Rails 7.1, but do not yet depend on Rack 3, due to a third-party dependency that doesn't support Rack >= 3. We also use [Tapioca](https://github.com/Shopify/tapioca) to generate RBI files. As part of the RBI generation process, the app dependencies are required, including `rackup`.

**Comment by cbochs** [comment]
I might recommend going so far as to pulling the `v1.0.0` release since it is obviously broken.

**Comment by joshgoebel** [comment]
And could we perhaps get a new release that's not broken?

**Comment by joshgoebel** [comment]
Ping.

**Comment by ioquatix** [comment]
Sorry, I have not been getting any notifications. Probably best to mention someone by name in the future if you aren't getting any traction.

Let me take a look at what's gone wrong here.

**Comment by ioquatix** [comment]
Okay, this should be fixed in v1.0.1

Yes, it's intended as a "fake" release for Rack 2.x - otherwise adding `rackup` as a gem (intended for Rack 3) will cause problems when used with Rack 2.
